<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAnotationManager</class>
 <widget class="QMainWindow" name="WindowAnotationManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBrowse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listFiles">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Current tracker</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextEdit" name="textEditTrackerInfo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAnotate">
          <property name="text">
           <string>Anotate</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/pencil.png</normaloff>:/icons/icons/pencil.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTrack">
          <property name="text">
           <string>Track</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/eye.png</normaloff>:/icons/icons/eye.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCoordSystem">
          <property name="text">
           <string>Coord system</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStats">
          <property name="text">
           <string>Strain statistics</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/chart_curve.png</normaloff>:/icons/icons/chart_curve.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBeatToBeat">
          <property name="text">
           <string>Beat to beat variance</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="WidgetStrainVideo" name="widgetStrainVideo" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxBeats">
      <property name="title">
       <string>Beats</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QListView" name="listViewBeats">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnAddBeat">
           <property name="text">
            <string>&amp;Add Current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveBeat">
           <property name="text">
            <string>&amp;Remove selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuChange_tracker">
     <property name="title">
      <string>Change tracker</string>
     </property>
     <widget class="QMenu" name="menuPoint_Tracker">
      <property name="title">
       <string>Point Tracker</string>
      </property>
      <addaction name="actionSAD"/>
      <addaction name="actionSSD"/>
      <addaction name="actionCorrelation"/>
      <addaction name="actionCosine"/>
      <addaction name="actionOptical_flow"/>
      <addaction name="actionOptical_flow_with_neighbourhood"/>
     </widget>
     <widget class="QMenu" name="menuWeights">
      <property name="title">
       <string>Weights</string>
      </property>
      <addaction name="actionWeights1"/>
      <addaction name="actionWeights2"/>
      <addaction name="actionWeights3"/>
      <addaction name="actionWeights4"/>
     </widget>
     <widget class="QMenu" name="menuResut_Processing">
      <property name="title">
       <string>Resut Processing</string>
      </property>
      <addaction name="actionResProcNone"/>
      <addaction name="actionResProcAvg"/>
     </widget>
     <addaction name="menuPoint_Tracker"/>
     <addaction name="menuWeights"/>
     <addaction name="menuResut_Processing"/>
    </widget>
    <addaction name="menuChange_tracker"/>
    <addaction name="actionChangeImageProcessing"/>
    <addaction name="actionShowShapeModel"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChangeImageProcessing">
   <property name="text">
    <string>Change image processing</string>
   </property>
  </action>
  <action name="actionShowShapeModel">
   <property name="text">
    <string>Show statistical shape model</string>
   </property>
  </action>
  <action name="actionSAD">
   <property name="text">
    <string>SAD</string>
   </property>
  </action>
  <action name="actionSSD">
   <property name="text">
    <string>SSD</string>
   </property>
  </action>
  <action name="actionOptical_flow">
   <property name="text">
    <string>Optical flow</string>
   </property>
  </action>
  <action name="actionOptical_flow_with_neighbourhood">
   <property name="text">
    <string>Optical flow with neighbourhood</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionPCA">
   <property name="text">
    <string>PCA</string>
   </property>
  </action>
  <action name="actionCorrelation">
   <property name="text">
    <string>Correlation</string>
   </property>
  </action>
  <action name="actionCosine">
   <property name="text">
    <string>Cosine</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionWeights1">
   <property name="text">
    <string>1.0</string>
   </property>
  </action>
  <action name="actionWeights2">
   <property name="text">
    <string>1.0 0.5</string>
   </property>
  </action>
  <action name="actionWeights3">
   <property name="text">
    <string>1.0 0.5 0.25</string>
   </property>
  </action>
  <action name="actionWeights4">
   <property name="text">
    <string>1.0 0.5 0.25 0.125</string>
   </property>
  </action>
  <action name="actionResProcNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionResProcAvg">
   <property name="text">
    <string>Floating average</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetStrainVideo</class>
   <extends>QWidget</extends>
   <header>widgetstrainvideo.h</header>
   <container>1</container>
   <slots>
    <signal>displayIndexChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_actionCreateTracker()</slot>
  <slot>on_actionChangeImageProcessing()</slot>
  <slot>on_actionShowShapeModel()</slot>
 </slots>
</ui>
